src/schoology-extractor/edfi_schoology_extractor/client_facade.py

```diff
@@ -27,7 +27,7 @@
 
 
 @dataclass
-class SchoologyExtractFacade:
+class ClientFacade:
```

Comment:
In a prior ticket I changed the name and created a new `extract_facade.py`. But I overlooked changing the class name. So I took care of it here.